index.js

```diff
@@ -206,7 +206,7 @@ function checked (length) {
     throw new RangeError('Attempt to allocate Buffer larger than maximum ' +
                          'size: 0x' + kMaxLength.toString(16) + ' bytes')
   }
-  return length >>> 0
+  return length | 0
```

Comment:
Why? I'd rather keep this the same as the iojs codebase so it's easier to stay in sync.